manifests/quick-start-postgres.yaml

```diff
@@ -156,7 +156,12 @@ spec:
         type: object
     served: true
     storage: true
-    subresources: {}
+    subresources:
+      scale:
+        labelSelectorPath: .status.selector
+        specReplicasPath: .spec.parallelism
+        statusReplicasPath: .status.parallelism
+      status: {}
```

Comment:
Good to see this status subresource here. Would you like to update the PR title to reflect this change too?

Reply:
For sure! Thanks for the suggestion @terrytangyuan

Reply:
Including the status subresource seems to break everything. I'm going to remove it for now 😞